src/Makefile

```diff
@@ -0,0 +1,18 @@
+# the compiler: gcc for C program, define as g++ for C++
+CC = gcc
+
+# compiler flags:
+#  -g    adds debugging information to the executable file
+#  -Wall turns on most, but not all, compiler warnings
+CFLAGS = -g -Wall
+
+# the build target executable:
+TARGET = code
```

Comment:
This line is the most important because it defines the target.
Think about how to make this more generic if other code is added to the project.